tests/js/client/shell/shell-collection-properties-cluster-spec.js

```diff
@@ -40,7 +40,7 @@ const cn2 = "UnitTestPropertiesFollower";
 // check whether all shards have the right amount of followers
 function checkReplicationFactor(name, fac) {
     // first we need the plan id of the collection
-    let plan = ArangoAgency.get('Plan/Collections/_system');
+    let plan = arango.POST("/_admin/execute", `return global.ArangoAgency.get('Plan/Collections/_system')`);
```

Comment:
you can at least fetch the full plan and work on it. No need for remote js imho.